scene/2d/physics_body_2d.cpp

```diff
@@ -1211,9 +1211,9 @@ void KinematicBody2D::_notification(int p_what) {
 		Transform2D new_transform = get_global_transform();
 		PhysicsServer2D::get_singleton()->body_set_state(get_rid(), PhysicsServer2D::BODY_STATE_TRANSFORM, new_transform);
 		//but then revert changes
-		set_notify_local_transform(false);
-		set_global_transform(last_valid_transform);
-		set_notify_local_transform(true);
+		// set_notify_local_transform(false);
+		// set_global_transform(last_valid_transform);
+		// set_notify_local_transform(true);
```

Comment:
Commenting on the code is generally not desired, usually, you should remove it completely if it's not used.